src/contrib/cluster/Akka.DistributedData/DeltaPropagationSelector.cs

```diff
@@ -68,13 +69,11 @@ public void Delete(string key)
                 if (all.Length <= sliceSize) slice = all;
                 else
                 {
-                    var start = (int)(_deltaNodeRoundRobinCounter % all.Length);
```

Comment:
This part of the code also looks suspicious, so I reported it with a cleaner one.
Reference scala code:
```scala
val i = (deltaNodeRoundRobinCounter % all.size).toInt
val first = all.slice(i, i + sliceSize)
if (first.size == sliceSize) first
else first ++ all.take(sliceSize - first.size)
```